test/e2e/tests/solana/web-socket-connection.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { strict as assert } from 'assert';
+import { Suite } from 'mocha';
+import {
+  withSolanaAccountSnap,
+  getWebsocketConnectionCount,
+} from './common-solana';
+import { TestDappSolana } from '../../page-objects/pages/test-dapp-solana';
+import { DEFAULT_SOLANA_TEST_DAPP_FIXTURE_OPTIONS } from '../../flask/solana-wallet-standard/testHelpers';
+
+describe('Solana Web Socket', function (this: Suite) {
+  it('a websocket connection is open when MetaMask full view is open', async function () {
+    await withSolanaAccountSnap(
+      {
+        title: this.test?.fullTitle(),
+      },
+      async (driver) => {
+        // Wait a moment for the websocket connection to be established
+        await driver.delay(3000);
```

Comment:
note: all the delays are intentional in this spec, to ensure proper validations without unexpected things happenbing after some seconds (no new connections or terminations).
But the normal Solana specs don't need any of these delays, neither for start nor terminate the WS server and the connection - which is expected